modules/ossm-about-external-control-plane-topology.adoc

```diff
@@ -0,0 +1,8 @@
+// This procedure is used in the following assembly:
+// * install/ossm-multi-cluster-topologies.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="ossm-about-external-control-plane-topology_{context}"]
+= About external control plane topology
```

Comment:
```suggestion
= About the external control plane topology
```

Reply:
Not a deal breaker for merge just making note for future content improvement. I saw you used `the` in the first sentence rather than `External...improves`.